### ISSUE TITLE
Message ID issue while sending sendFileFromUrl and Location

### ISSUE THREAD

**Issue by muzammilgit** [open-wa/wa-automate-nodejs#543]
I am unable to get message id when using sendFileFromUrl and location

**Comment by smashah** [comment]
Set the last Param true for sending file.

https://open-wa.github.io/wa-automate-nodejs/classes/client.html#sendfilefromurl

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [1.9.70](https://github.com/open-wa/wa-automate-nodejs/compare/1.9.69...1.9.70) (2020-06-13)

- Release 1.9.70 [`b024c15`](https://github.com/open-wa/wa-automate-nodejs/commit/b024c153f41af1b4ffbbcfbebd46a921757c83a6)
- feature: return message id on `sendLocation` #543 [`18e3d98`](https://github.com/open-wa/wa-automate-nodejs/commit/18e3d98c22837702cb341ca9ff9a5358c7bb884a)

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [1.9.73](https://github.com/open-wa/wa-automate-nodejs/compare/1.9.72...1.9.73) (2020-06-14)

- Release 1.9.73 [`8c83bb0`](https://github.com/open-wa/wa-automate-nodejs/commit/8c83bb0e9b472cb67e02b03a51cd134382b47af3)
- update docs and fix sendLcation #543 [`16d42bc`](https://github.com/open-wa/wa-automate-nodejs/commit/16d42bc8450284c5359e159b761b32a68f9969b9)

**Comment by smashah** [comment]
@muzammilgit it is working now
